MovieGPT-api/src/main/java/lt/techin/group/project/exception/GlobalExceptionHandler.java

```diff
@@ -132,4 +133,24 @@ public ResponseEntity<ErrorResponse> handleEmailAlreadyExistsException(EmailAlre
         );
         return ResponseEntity.status(HttpStatus.CONFLICT).body(errorResponse);
     }
+
+    @ExceptionHandler(CommentNotFoundException.class)
+    public ResponseEntity<ErrorResponse> handleUserNotFoundException(CommentNotFoundException ex) {
+        ErrorResponse errorResponse = new ErrorResponse(
+                HttpStatus.NOT_FOUND.value(),
+                NOTFOUND,
+                ex.getMessage()
+        );
+        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(errorResponse);
+    }
+
+    @ExceptionHandler(CannotEditAnotherUserException.class)
+    public ResponseEntity<ErrorResponse> handleCannotEditAnotherUserException(CannotEditAnotherUserException ex) {
```

Comment:
I have absolutely nothing to comment on as code seems great, but if I really wanted to say smth, I would maybe offer to rename this exception to CannotEditAnotherUserComment to avoid ambiguity what we are editing. :)

Reply:
Thanks, forgot one word. Will be fixed in another push with more little improvements.